src/main/java/com/vitalii/twenty/methods/TwentyMethods.java

```diff
@@ -24,4 +24,14 @@ public static int returnMaxOfTwoInts(int a, int b) {
         }
         return b;
     }
+
+    //3.1 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
+    public static int returnMaxOfTwoIntsWithMath(int c, int d) {
+        return Math.max(c, d);
+    }
+
+    //3.2 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
```

Comment:
Instead // use /** .... */ in code. It is comment too, but look like documentation when you click on pen icon in left side.

Reply:
to be fixed in next pull request